app/controllers/rentals_controller.rb

```diff
@@ -6,29 +6,27 @@ def index
     @rentals = Rental.where(user_id: current_user.id)
   end
 
-  def new
-    @rental = Rental.new
-    @book = Book.find(params[:book_id])
-    authorize @rental
-  end
-
   def show
   end
 
   def create
     @rental = Rental.new(rental_params)
-    @rental.user = current_user
-    @rental.book = Book.find(params[:book_id])
-
     authorize @rental
-
+    @rental.user = current_user
     if @rental.save!
-      redirect_to root_path
```

Comment:
please just don't accept the changes in the rental file -  i did not make them, I think they are a merging error